Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -707,25 +714,26 @@ extension FirefoxHomeViewController: UICollectionViewDelegateFlowLayout {
             }
             return cellSize
         case .libraryShortcuts:
-            let width = min(FirefoxHomeUX.LibraryShortcutsMaxWidth, cellSize.width)
+            let width = min(FirefoxHomeUX.libraryShortcutsMaxWidth, cellSize.width)
             return CGSize(width: width, height: cellSize.height)
         case .customizeHome, .pocket, .recentlySaved:
             return cellSize
         }
     }
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+
         switch Section(section) {
         case .pocket:
-            return pocketStories.isEmpty ? .zero : Section(section).headerHeight
+            return pocketStories.isEmpty ? .zero : getHeaderSize(forSection: section)
         case .topSites:
-            return isTopSitesSectionEnabled ? Section(section).headerHeight : .zero
+            return isTopSitesSectionEnabled ? getHeaderSize(forSection: section) : .zero
         case .libraryShortcuts:
-            return isYourLibrarySectionEnabled ? Section(section).headerHeight : .zero
+            return isYourLibrarySectionEnabled ? getHeaderSize(forSection: section) : .zero
         case .jumpBackIn:
-            return isJumpBackInSectionEnabled ? Section(section).headerHeight : .zero
+            return isJumpBackInSectionEnabled ? getHeaderSize(forSection: section) : .zero
         case .recentlySaved:
-            return isRecentlySavedSectionEnabled ? Section(section).headerHeight : .zero
+            return isRecentlySavedSectionEnabled ? getHeaderSize(forSection: section) : .zero
```

Comment:
This is great! I would also double check: are we using `Section(section).headerHeight` anywhere else? If not, can we get rid of it?

Reply:
Yessir, we can get rid of it! It's in the PR 😇
![Screen Shot 2021-11-01 at 4 56 18 PM](https://user-images.githubusercontent.com/11338480/139741085-97b204c0-f496-4753-a40a-b5b35a97871f.png)

Reply:
I obviously did not fully pay attention and forgot. Foot in mouth!